# AndroidLogEntry Message

std_msgs/Header header

std_msgs/UInt8 id
std_msgs/UInt64 ts
std_msgs/UInt8 priority
std_msgs/Int32 pid
std_msgs/Int32 tid
std_msgs/String tag
std_msgs/String message



# struct AndroidLogEntry {
#   id @0 :UInt8;
#   ts @1 :UInt64;
#   priority @2 :UInt8;
#   pid @3 :Int32;
#   tid @4 :Int32;
#   tag @5 :Text;
#   message @6 :Text;
# }
